lib/mayaUsd/fileio/primUpdaterManager.h

```diff
@@ -103,14 +153,64 @@ class PrimUpdaterManager : public PXR_NS::TfWeakBase
     //! Create the pull parent and set it into the prim updater context.
     MDagPath setupPullParent(const Ufe::Path& pulledPath, VtDictionary& args);
 
+    //! Record pull information for the pulled path, for inspection on
+    //! scene changes.
+    void recordPullVariantInfo(const Ufe::Path& pulledPath, const MDagPath& pullParentPath);
+
+    // Maya file new or open callback.  Member function to access other private
+    // member functions.
+    static void beforeNewOrOpenCallback(void* clientData);
+
+    void beginManagePulledPrims();
+    void endManagePulledPrims();
+
+    // Member function to access private nested classes.
+    static std::list<VariantSetDescriptor> variantSetDescriptors(const Ufe::Path& path);
```

Comment:
Minor: having a class named VariantSetDescriptors and a function named variantSetDescriptors is a bit confusing. Maybe a "get"or "compute" prefix on the function?